inst/rsn/RSN_02.Rmd

```diff
@@ -0,0 +1,390 @@
+## "zero-area" polygons. Run this chunk again if it doesn't work.
+
+lc_eth_spdf<-allpts
+
+par(mar=c(0,0,0,0), oma=c(0,0,0,0))
+plot(voronoi, main=NULL);
+plot(lc_eth_spdf, add=TRUE, main=NULL)
+
+```
+
+### FIX--FIGURE OUT HOW TO HANDLE POINTS IN ETHIOPIA/ERITREA BORDER 
+
+```{r hh_match}
+
+missingrows<-lc_ethiopia[setdiff(lc_ethiopia$id, lc_eth_spdf$newid),]
+missingrows<-SpatialPoints(data.frame(missingrows$longitude, missingrows$latitude))
+
+plot(ethiopia); plot(missingrows, add=TRUE, col="red", fil="red", pch=16, cex=1)
+
+# There is one point that is outside the border of Ethiopia. This is part of a contested boundary--need to figure out how to process
+
+```
+
+### Merge survey data with spatial data
+
+```{r merge_surv_spat}
+
+lc_eth_spdf@data<-merge(lc_eth_spdf@data, lc_ethiopia, 
+                        by.x="newid", by.y="id", all=FALSE)
+
+```
+
+### Initial plots
+
+```{r plot1, fig.height=10, fig.width=8}
+
+# Create a layer-list
+eth_outline <- list("sp.polygons", ethiopia, col = "grey")
+
+par(oma=c(0,0,0,0), mar=c(0,0,0,0))
+
+pal1 <- brewer.pal(n =4, name = "Reds"); 
+chickens<-spplot(lc_eth_spdf, "chicken", 
+       cuts = quantile(lc_eth_spdf$chicken[lc_eth_spdf$chicken>0]),
+       col.regions = pal1, col = "transparent",
+       sp.layout = eth_outline, auto.key=TRUE, key.space="right",
+       par.settings = list(axis.line = list(col =  'transparent')),
+       main="Chickens")
+
+pal2 <- brewer.pal(n =4, name = "Greens"); 
+cattle<-spplot(lc_eth_spdf, "cattle", 
+       cuts = quantile(lc_eth_spdf$cattle[lc_eth_spdf$cattle>0]),
+       col.regions = pal2, col = "transparent",
+       sp.layout = eth_outline, auto.key=TRUE, key.space="right",
+       par.settings = list(axis.line = list(col =  'transparent')),
+       main="Cattle")
+
+
+pal3 <- brewer.pal(n=4, name = "Greys"); 
+sheep<-spplot(lc_eth_spdf, "sheep", 
+       cuts = quantile(lc_eth_spdf$sheep[lc_eth_spdf$sheep>0]),
+       col.regions = pal3, col = "transparent",
+       sp.layout = eth_outline, auto.key=TRUE, key.space="right",
+       par.settings = list(axis.line = list(col =  'transparent')),
+       main="Sheep")
+
+pal4 <- brewer.pal(n=4, name = "YlOrBr"); 
+goats<-spplot(lc_eth_spdf, "goats", 
+       cuts = quantile(lc_eth_spdf$goats[lc_eth_spdf$goats>0]),
+       col.regions = pal4, col = "transparent",
+       sp.layout = eth_outline, auto.key=TRUE, key.space="right",
+       par.settings = list(axis.line = list(col =  'transparent')),
+       main="Goats")
+
+pal5 <- brewer.pal(n=2, name = "Blues"); 
+otheran<-spplot(lc_eth_spdf, "otheran", 
+       cuts = quantile(lc_eth_spdf$otheran[lc_eth_spdf$otheran>0])[3:5],
+       col.regions = pal5, col = "transparent",
+       sp.layout = eth_outline, auto.key=TRUE, key.space="right",
+       par.settings = list(axis.line = list(col =  'transparent')),
+       main="Other Animals")
+
+grid.arrange(chickens, cattle, sheep, goats, otheran, ncol=2, nrow=3)
+
+```
+
```

Comment:
I think we're going to need some much simplified summary data on this data (some of which may already be provided, but should be summarized alongside the network stats), on number of households, distribution of farm sizes, number of markets and households served per market.